test/inlining/rlexe.xml

```diff
@@ -274,4 +274,10 @@
       <compile-flags>-loopinterpretcount:1 -bgjit- -force:inline</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bug9936017.js</files>
+      <compile-flags>-off:simplejit -oopjit-</compile-flags>
```

Comment:
Adding a third call to test1 removes the need for -off:simplejit, but I can't seem to get the test to fail without -oopjit-. It takes much longer to run and eventually prints "Passed" regardless of the number of calls to test1. Is there something I am missing?

---
In reply to: [122547975](https://github.com/Microsoft/ChakraCore/pull/3173#discussion_r122547975) [](ancestors = 122547975)